.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ commands:
     parameters:
       version:
         type: string
-        default: "1.15"
+        default: "1.16"
```

Comment:
The only thing that actually ended up using Go 1.16 was one place ware it said `io.Discard` instead of the older `ioutil.Discard`.